src/http-spec.ts

```diff
@@ -7,8 +7,58 @@ import { IServer } from './servers';
  * HTTP Service
  */
 
+interface IOAuthFlow {
+  authorizationUrl: string;
+  tokenUrl: string;
+  refreshUrl?: string;
+  scopes: Dictionary<string>;
+}
+
+type SecurityDefinition =
```

Comment:
Maybe we can leverage `HttpSecurityScheme` instead of this? To unify the scheme's stored on `IHttpOperation` and `IHttpService`?

Reply:
Good point, they actually match.